certified-connectors/tyntec-SMS/apiDefinition.swagger.json

```diff
@@ -235,6 +248,249 @@
           }
         ]
       }
+    },
+    "/conversations/v3/power-automate/messages/sms/text": {
+      "post": {
+        "responses": {
+          "202": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "messageId": {
+                  "type": "string",
+                  "description": "messageId",
+                  "x-ms-visibility": "important"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "SendSMSv3",
+        "summary": "Send a custom SMS",
+        "description": "Send a custom SMS",
```

Comment:
@ridlees we highly suggest you have different description than the summary. can you please change this?

Reply:
Changed